.github/workflows/pytest-workflow.yml

```diff
@@ -56,6 +56,8 @@ jobs:
             tags: concatenate_vcfs
           - profile: "conda"
             tags: sentieon/bwamem
+          - profile: "singularity"
+            tags: concatenate_vcfs
```

Comment:
what's happening there?

Reply:
Skipping singularity-based pytest `concatenate_vcfs` since it was failing. We did the same change on the `dev`-branch.

Reply:
yeah, but you should get that by syncing the `dev` branch, not by changing it yourself

Reply:
I know I'm being picky

Reply:
My plan is to get all test working on the PR, merge the PR (ie. `sentieon_haplotyper` into `sentieon`), then merge `dev` into `sentieon` and deal with merge conflicts there.

Reply:
ok then